src/go/rpk/pkg/cli/cluster/partitions/list.go

```diff
@@ -30,6 +30,11 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+type response struct {
```

Comment:
So, this is our first time with a breaking change in our `--format` response in rpk. We should avoid breaking current users where we can, but here, I think it's inevitable. This should prevent us from backporting to older branches.

It's fine, but we shouldn't backport and add this to the release notes. In retrospect, I shouldn't have returned an array in the first place, but instead an array as part of an object `{partitions:[]rpadmin.ClusterPartition}`

w.d.y.t @twmb

Reply:
It's ok to have this feature since 25.1.